src/domain/stats/useVolumeInfo.ts

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from "react";
+import { gql } from "@apollo/client";
+import { getGmxGraphClient } from "lib/subgraph/clients";
+import { bigNumberify } from "lib/numbers";
+import { ARBITRUM, AVALANCHE } from "config/chains";
+import { BigNumber } from "ethers";
+
+type VolumeInfo = {
+  totalVolume: BigNumber;
+  [AVALANCHE]: { totalVolume: BigNumber };
+  [ARBITRUM]: { totalVolume: BigNumber };
+};
+
+type VolumeStat = {
+  swap: string;
+  margin: string;
+  liquidation: string;
+  mint: string;
+  burn: string;
+};
+
+export default function useVolumeInfo() {
+  const [volumeInfo, setVolumeInfo] = useState<null | VolumeInfo>(null);
+  useEffect(() => {
```

Comment:
You can wrap this to useSWR to not store the state locally

```
 const { data, error } = useSWR<VolumeInfo[]>(['useVolumeInfo'], {
    fetcher: async () => {
      const query = gql(`{
      }`);

      const { data } = await client!.query({ query });

      ... processing
      
     return volumeInfo
    },
  });
```

Reply:
discussed and decided not to, since in other places subgraphs requests don't use `useSWR`